Testing/Exec/IgnitionDelay/check_ignDelay.py

```diff
@@ -18,7 +18,7 @@ def readResult(filename):
     tIgn,uncertainty = readResult('log')
 
     # Compare to theoretical value
-    if abs(tIgn-0.0763)>10*uncertainty:
+    if abs(tIgn-0.0763)>15*uncertainty:
```

Comment:
I think you are doing this because you want the test to pass with the manifold.

Could you please maybe change check_ignDelay to receive 2 arguments (with argparse): 1 is the expected ignition delay (0.0763) and 1 would the tolerance? Then you can change the tolerance for the manifold test.

Reply:
Done.